app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -175,10 +178,15 @@ acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
   });
 
   test("click event not fired twice when both dismiss buttons are present", async function (assert) {
-    // visit root first so topic list starts fresh
+    // need to scroll to top so the viewport shows the top of the page
+    // and top dismiss button
     await visit("/");
+    await (document.getElementById("ember-testing-container").scrollTop = 0);
```

Comment:
```suggestion
    document.getElementById("ember-testing-container").scrollTop = 0;
```

(I'm making all three comments in case you want click-to-commit 😝)